File: Studiopro/src/app/models/admin/company/CompanyAddress.ts
interface CompanyAddress {
    CompanyAddressId: number;

    CompanyId: number;

    Label: string;

    Address: string;

    Area: string | null;

    City: string;

    State: string;

    Pincode: string;

    Landmark: string | null;

    IsPrimary: boolean;

    Status: number;

    StatusName: string;

    CreatedBy: number;

    CreatedByName: string;

    CreatedDate: Date | string;

    ModifiedBy: number | null;

    ModifiedByName: string;

    ModifiedDate: Date | string | null;
}

File: Service/Models/CompanyAddress.cs
using System;
using System.Collections.Generic;

namespace Service.Models;

public partial class CompanyAddress
{
    public int CompanyAddressId { get; set; }

    public int CompanyId { get; set; }

    public string Label { get; set; } = null!;

    public string Address { get; set; } = null!;

    public string? Area { get; set; }

    public string City { get; set; } = null!;

    public string State { get; set; } = null!;

    public string Pincode { get; set; } = null!;

    public string? Landmark { get; set; }

    public bool IsPrimary { get; set; }

    public int Status { get; set; }

    public int CreatedBy { get; set; }

    public DateTime CreatedDate { get; set; }

    public int? ModifiedBy { get; set; }

    public DateTime? ModifiedDate { get; set; }

    public virtual Company Company { get; set; } = null!;
}
